Added Additional Test to Increment a Number

jnmorse opened the pull request:
Added an additional test to check that myVar = myVar is not still
present.  Tested locally.

closes #7705

raisedadead commented on seed/challenges/01-front-end-development-certification/basic-javascript.json:
```diff
@@ -3458,6 +3456,9 @@
         "assert(code.match(/break/g).length > 2, 'message: You should have at least 3 <code>break</code> statements');"
       ],
       "type": "waypoint",
+      "MDNlinks": [
```
@jnmorse I am not sure if this is accidental, but this diff looks weird. Did you move this?

jnmorse replied:
@raisedadead I had edited with COM1000 for some reason that line was moved when it was saved.

jnmorse replied:
As a note, I had left it cause it didn't seem to effect anything else, just the position in the file.  When I worked on the pull request I submitted after this one though #7863, the entire file had been re-ordered so I went back and manually changed that one, since the diff would have been the entire file.

raisedadead replied:
Well in that case if you can rebase then its good. :confused: 
Or I would recommend making a fresh branch of the latest staging and adding your test case.

Just make sure you save your fix somewhere before hand.
I am sure you have gone through this but just in case : [How to maintain a local instance](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/How-To-Fork-And-Maintain-a-Local-Instance-of-Free-Code-Camp)

Not sure if this was because of COM1000, dun have much idea on how it works.

ghostsleeper7 commented on seed/challenges/01-front-end-development-certification/basic-javascript.json:
```diff
@@ -443,6 +443,7 @@
       ],
       "tests": [
```
" faith(myVar\|p*\\=.*mylar ==)

jnmorse committed:
Added Additional Test to Increment a Number

Added an additional test to check that myVar = myVar is not still
present. Tested locally.

jnmorse commented:
@raisedadead Okay, so went ahead and reverted the commit, moved the code that COM1000 had moved back to its original location and then did force pushed the commit back up

raisedadead commented:
#### :sparkles: :sparkles: **PERFECTO.** :sparkles: :sparkles: 

Thanks @jnmorse you are awesome!
